### PR TITLE
Update the `bal help` CLI texts

### LINKED ISSUE #8536: Add the latest help command text in the example given.
## Description
$subject in https://ballerina.io/learn/cli-commands/#use-the-ballerina-tool

Following image is having older version of the command. 

image

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Update the `bal help` CLI texts.
> Fixes #8536 and #8537

## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: swan-lake/development-tutorials/build-and-run/cli-commands.md [hunk 1]
```diff
@@ -70,12 +71,20 @@ COMMANDS
         persist         Manage data persistence
         bindgen         Generate the Ballerina bindings for Java APIs
         shell           Run Ballerina interactive REPL [Experimental]
+        tool            Manage Ballerina CLI tools
         version         Print the Ballerina version
         profile         Start Ballerina Profiler [Experimental]
 
+   Tool Commands:
+        edi             Generates Ballerina service/client for OpenAPI contract and
```
Comment (praneesha) [inline swan-lake/development-tutorials/build-and-run/cli-commands.md:79]:
```suggestion
        edi             Generates Ballerina service/client for OpenAPI contract and
```
@RDPerera - Can you send the short description for the EDI tool?

**Review by gayaldassanayake — DISMISSED** [review]

**Comment by gayaldassanayake** [comment]
This PR addresses the issue of listing the `Tool Command` under the wrong title as mentioned in https://github.com/ballerina-platform/ballerina-lang/issues/41980 and https://github.com/ballerina-platform/ballerina-dev-website/issues/8535

**Comment by praneesha** [comment]
Updated the GraphQL description also as per https://github.com/ballerina-platform/ballerina-lang/pull/41983/files.

**Review by ThisaruGuruge — DISMISSED** [review]
GraphQL CLI text LGTM. @DimuthuMadushan any thoughts?

**Review by gayaldassanayake — DISMISSED** [review]
graphql and edi short help texts needs to be updated in the lang repo to match the descriptions here.

File: swan-lake/development-tutorials/build-and-run/cli-commands.md [hunk 2]
```diff
@@ -61,21 +62,26 @@ COMMANDS
         format          Format Ballerina source files
         grpc            Generate the Ballerina sources for a given Protocol
                         Buffer definition
-        graphql         Generate the Ballerina client sources for a GraphQL config file,
-                        generate the GraphQL schema for a Ballerina GraphQL service, and 
-                        generate the Ballerina GraphQL service for a GraphQL schema
+        graphql         Generate the Ballerina client sources for a GraphQL config file, the GraphQL schema for a GraphQL service, and Ballerina service sources for a GraphQL schema
```
Comment (DimuthuMadushan) [inline swan-lake/development-tutorials/build-and-run/cli-commands.md:65]: It seems this line is longer than others. Shall we break this line? (like in OpenApi)

**Review by RDPerera — DISMISSED** [review]
EDI Tool related changes are LGTM

File: swan-lake/development-tutorials/build-and-run/cli-commands.md (hunk 2 above)
Comment (DimuthuMadushan) [reply swan-lake/development-tutorials/build-and-run/cli-commands.md:65]: IIRC there is a character limit per line. @praneesha can check this?

Comment (praneesha) [reply swan-lake/development-tutorials/build-and-run/cli-commands.md]: Updated.

**Review by DimuthuMadushan — COMMENTED** [review]
GraphQL CLI text LGTM

File: swan-lake/development-tutorials/build-and-run/cli-commands.md [hunk 3]
```diff
@@ -61,21 +62,26 @@ COMMANDS
...
         persist         Manage data persistence
         bindgen         Generate the Ballerina bindings for Java APIs
         shell           Run Ballerina interactive REPL [Experimental]
+        tool            Manage Ballerina CLI tools
         version         Print the Ballerina version
         profile         Start Ballerina Profiler [Experimental]
 
+   Tool Commands:
+        edi             Generate the Ballerina records and parsing functions or a Ballerina package for a given EDI schema or collection of schemas
+        health          Ballerina Health Artifact Generator Tool
```
[the remark this answers is not in the document]
Comment (isuruh15) [reply swan-lake/development-tutorials/build-and-run/cli-commands.md:80]: Shall we update like, `Generate the Ballerina packages or API templates for a given FHIR Implementation guide.`

[the remark this answers is not in the document]
Comment (praneesha) [reply swan-lake/development-tutorials/build-and-run/cli-commands.md]: Updated.

**Comment by nirmal070125** [comment]
Looks good.

**Review by gayaldassanayake — APPROVED** [review]

**Pull request merged**